### PR TITLE
MODCLUSTER-482 Drop Tomcat 6 and JBoss Web support

### PR DESCRIPTION
**Pull request by rhusar**
Jiras
https://issues.jboss.org/browse/MODCLUSTER-482
https://issues.jboss.org/browse/MODCLUSTER-479

Next up
https://issues.jboss.org/browse/MODCLUSTER-439 revamp distribution profile
https://issues.jboss.org/browse/MODCLUSTER-502 using JDK7 channel APIs

### CODE REVIEW

**Comment by modcluster-pull-request** [comment]
Triggering build using a merge of 0ec3633b2c546668937ceaa75508eda9cc5decf0 on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/mod_cluster-1.3.x-pull-player-executor/

**Comment by modcluster-pull-request** [comment]
Build 188 outcome was SUCCESS using a merge of 0ec3633b2c546668937ceaa75508eda9cc5decf0 on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/mod_cluster-1.3.x-pull-player-executor/188
Public: https://hudson.jboss.org/hudson/job/mod_cluster-1.3.x-pull-player-executor/188

**Comment by modcluster-pull-request** [comment]
Triggering build using a merge of 808b8e5bdc246015e4f99f8bc186f30715040150 on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/mod_cluster-1.3.x-pull-player-executor/

**Comment by modcluster-pull-request** [comment]
Build 192 outcome was SUCCESS using a merge of 808b8e5bdc246015e4f99f8bc186f30715040150 on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/mod_cluster-1.3.x-pull-player-executor/192
Public: https://hudson.jboss.org/hudson/job/mod_cluster-1.3.x-pull-player-executor/192

**Comment by modcluster-pull-request** [comment]
Triggering build using a merge of 308a43097c835600e337b23a8f3bf4ce7e5c7ba7 on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/mod_cluster-1.3.x-pull-player-executor/

**Comment by modcluster-pull-request** [comment]
Build 193 outcome was SUCCESS using a merge of 308a43097c835600e337b23a8f3bf4ce7e5c7ba7 on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/mod_cluster-1.3.x-pull-player-executor/193
Public: https://hudson.jboss.org/hudson/job/mod_cluster-1.3.x-pull-player-executor/193

File: README.md [hunk 1]
```diff
@@ -53,10 +53,7 @@ Project Structure
 ```
 container-spi (contains no dependencies on a specific web container)
 container
-  catalina (base Tomcat/JBoss Web container implementation, based on Tomcat 5.5)
-  catalina-standalone (contains ModClusterListener, used for standalone Tomcat/JBoss Web installations)
-  jbossweb (JBoss Web container implementation, all versions)
-  tomcat6 (Tomcat 6.0 container implementation)
+  catalina (base for Tomcat container implementations, based on Tomcat 7)
```
Comment (pferraro) [inline README.md]: This is called "tomcat" now, right?

File: README.md [hunk 2]
```diff
@@ -53,10 +53,7 @@ Project Structure
...
   tomcat7 (Tomcat 7.0 container implementation)
```
Comment (pferraro) [inline README.md]: This no longer exists?

File: README.md (hunk 1 above)
Comment (rhusar) [inline README.md]: Done..

File: README.md (hunk 2 above)
Comment (rhusar) [inline README.md]: Yes, removed. Although I intended to rework this afterwards as part of MODCLUSTER-439.

**Comment by modcluster-pull-request** [comment]
Triggering build using a merge of 7bbee707b91955a867bb110f8b50918bf1d8c8c8 on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/mod_cluster-1.3.x-pull-player-executor/

**Comment by modcluster-pull-request** [comment]
Build 194 outcome was SUCCESS using a merge of 7bbee707b91955a867bb110f8b50918bf1d8c8c8 on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/mod_cluster-1.3.x-pull-player-executor/194
Public: https://hudson.jboss.org/hudson/job/mod_cluster-1.3.x-pull-player-executor/194

**Comment by pferraro** [comment]
Can we move "container-spi" to container/spi ?

File: container/tomcat/src/main/java/org/jboss/modcluster/container/catalina/standalone/ModClusterListener.java [hunk 3]
```diff
@@ -19,12 +19,13 @@
...
 
 /**
- * @author Paul Ferraro
+ * Compatibility class.
  *
+ * @author Radoslav Husar
+ * @version Jun 2016
  */
-public class ServerTestCase extends org.jboss.modcluster.container.catalina.ServerTestCase {
+public class ModClusterListener extends org.jboss.modcluster.container.tomcat.ModClusterListener {
```
Comment (pferraro) [inline container/tomcat/src/main/java/org/jboss/modcluster/container/catalina/standalone/ModClusterListener.java]: This should be @Deprecated

Comment (jfclere) [inline container/tomcat/src/main/java/org/jboss/modcluster/container/catalina/standalone/ModClusterListener.java]: OK for the @Deprecated ;-)
But changing "container-spi" to container/spi should go in another PR this one is already huge, no?

Comment (rhusar) [inline container/tomcat/src/main/java/org/jboss/modcluster/container/catalina/standalone/ModClusterListener.java]: Done.

**Comment by modcluster-pull-request** [comment]
Triggering build using a merge of c704b371cdcf008225160fd46a28f90c4f7b683d on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/mod_cluster-1.3.x-pull-player-executor/

**Comment by rhusar** [comment]
> Can we move "container-spi" to container/spi ?

Yes we can, I can't think of any harm. Let's do that in subsequent PR.

**Comment by modcluster-pull-request** [comment]
Build 197 outcome was SUCCESS using a merge of c704b371cdcf008225160fd46a28f90c4f7b683d on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/mod_cluster-1.3.x-pull-player-executor/197
Public: https://hudson.jboss.org/hudson/job/mod_cluster-1.3.x-pull-player-executor/197
